components/app_bar/app_bar.scss

```diff
@@ -0,0 +1,113 @@
+$app-bar-icon-size: 24px;
+$app-bar-width: 48px;
+$channel-view-padding: 63px;
+// We don't have control over the class names on the plugin channel header components, so we cast a net on the known variants.
+$app-bar-plugin-component: '.fa, i, svg';
+
+.app-bar-enabled .sidebar--right {
+    @media screen and (min-width: 769px) {
+        right: $app-bar-width;
+    }
+}
+
+.channel-view-inner.app-bar-enabled {
+    @media screen and (min-width: 769px) {
+        padding-right: $channel-view-padding;
+    }
+}
+
+.app-bar {
+    // Do not show App Bar if on mobile
```

Comment:
Do we have a plan for this? Will they (for the short term) still show as channel menu items?